devicetypes/smartthings/zigbee-window-shade.src/zigbee-window-shade.groovy

```diff
@@ -206,6 +218,10 @@ def configure() {
 		cmds += readDeviceBindingTable()
 	}
 
+	if (reportsBatteryPercentage()) {
+		cmds += zigbee.configureReporting(zigbee.POWER_CONFIGURATION_CLUSTER, BATTERY_PERCENTAGE_REMAINING, DataType.UINT8, 30, 720, 0x01)
```

Comment:
We typically go with a max report interval of 21600 for BATTERY_PERCENTAGE_REMAINING. Was there a particular reason you went lower?

Reply:
No particular reason. Let's go with typical value (21600)